jakarta-eclipse/src/org/jakarta/lsp4e/JakartaLanguageClient.java.orig

```diff
@@ -0,0 +1,137 @@
+package org.jakarta.lsp4e;
+
+import java.net.URI;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.CompletableFuture;
+
+import org.eclipse.core.runtime.IProgressMonitor;
+import org.eclipse.core.runtime.IStatus;
+import org.eclipse.core.runtime.NullProgressMonitor;
+import org.eclipse.core.runtime.Status;
+import org.eclipse.jdt.core.IAnnotation;
+import org.eclipse.jdt.core.ICompilationUnit;
+import org.eclipse.jdt.core.IImportDeclaration;
+import org.eclipse.jdt.core.IType;
+import org.eclipse.jdt.core.JavaModelException;
+import org.eclipse.jdt.core.IMethod;
+import org.eclipse.jdt.core.ISourceRange;
+import org.eclipse.lsp4e.LanguageClientImpl;
+import org.eclipse.lsp4j.Hover;
+import org.eclipse.lsp4j.HoverParams;
+import org.eclipse.lsp4j.MarkedString;
+import org.eclipse.lsp4j.PublishDiagnosticsParams;
+import org.eclipse.lsp4j.jsonrpc.CancelChecker;
+import org.eclipse.lsp4j.jsonrpc.CompletableFutures;
+import org.eclipse.lsp4j.jsonrpc.messages.Either;
+<<<<<<< HEAD
+import org.jakarta.jdt.JDTUtils;
+=======
+import org.jakarta.jdt.JDTServicesManager;
+>>>>>>> origin/master
+
+import io.microshed.jakartals.api.JakartaLanguageClientAPI;
+import io.microshed.jakartals.commons.JakartaDiagnosticsParams;
+
+public class JakartaLanguageClient extends LanguageClientImpl implements JakartaLanguageClientAPI {
+
+	public JakartaLanguageClient() {
+		// do nothing
+	}
+
+	private IProgressMonitor getProgressMonitor(CancelChecker cancelChecker) {
+		IProgressMonitor monitor = new NullProgressMonitor() {
+			public boolean isCanceled() {
+				cancelChecker.checkCanceled();
+				return false;
+			};
+		};
+		return monitor;
+	}
+
+	@Override
+	public CompletableFuture<Hover> getJavaHover(HoverParams params) {
+		// return dummy test hover object
+<<<<<<< HEAD
+		Activator.log(new Status(IStatus.INFO, "hover request received", "hover request received"));
+=======
+>>>>>>> origin/master
```

Comment:
These tags are left over from a rebase, let's rebase to resolve merge conflicts. Reach out if you have any questions